Examples/Framework/src/Framework/Sequencer.cpp

```diff
@@ -655,9 +656,17 @@ std::pair<std::string, std::size_t> Sequencer::fpeMaskCount(
     const boost::stacktrace::stacktrace& st, Acts::FpeType type) const {
   for (const auto& frame : st) {
     std::string loc = Acts::FpeMonitor::getSourceLocation(frame);
-    for (const auto& [filt, fType, count] : m_cfg.fpeMasks) {
-      if (boost::algorithm::ends_with(loc, filt) && fType == type) {
-        return {filt, count};
+    auto it = loc.find_last_of(':');
+    std::string locFile = loc.substr(0, it);
+    unsigned int locLine = std::stoi(loc.substr(it + 1));
+    for (const auto& [file, lines, fType, count] : m_cfg.fpeMasks) {
+      const auto [start, end] = lines;
+      if (boost::algorithm::ends_with(locFile, file) &&
+          (start <= locLine && locLine < end) && fType == type) {
+        std::string ls = start + 1 == end ? std::to_string(start)
+                                          : "(" + std::to_string(start) + ", " +
+                                                std::to_string(end) + "]";
+        return {file + ":" + ls, count};
       }
     }
```

Comment:
maybe this is worth an utility parsing function

nitpick: `unsigned int`

Reply:
I think this **is** the utility parsing function 😅